src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/IcebergColumnTypeMapper.java

```diff
@@ -11,13 +11,61 @@
 
 import com.snowflake.kafka.connector.internal.streaming.schemaevolution.ColumnTypeMapper;
 import net.snowflake.client.jdbc.internal.fasterxml.jackson.databind.JsonNode;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
 import org.apache.kafka.connect.data.Date;
 import org.apache.kafka.connect.data.Decimal;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.Time;
 import org.apache.kafka.connect.data.Timestamp;
 
 public class IcebergColumnTypeMapper extends ColumnTypeMapper {
+
+  /**
+   * See <a href="https://docs.snowflake.com/en/user-guide/tables-iceberg-data-types">Data types for
```

Comment:
see https://docs.snowflake.com/en/user-guide/tables-iceberg-data-types for details